docs/howto/create-a-fraction/index.adoc

```diff
@@ -83,22 +83,30 @@ a fraction contributes configuration or capabilities to a runtime system.
 
 ### Package Layout
 
-For a given fraction, a unique package root is required.  In the usual
-case of the core code, it matches the pattern of `org.wildfly.swarm._CAPABILITY_`,
+For a given fraction, a unique package root is required. 
+In the usual case of the core code, it matches the pattern of `org.wildfly.swarm._CAPABILITY_`,
 such as `org.wildfly.swarm.undertow` or `org.wildfly.swarm.naming`.
 
-Within the root may be two (or more) additional sub-packages, named
-'runtime' and 'deployment'.
+Within the root there may be additional sub-packages with special meaning:
 
-The `org.wildfly.swarm._CAPABILITY_.runtime` package holds classes that are
+* The `org.wildfly.swarm._CAPABILITY_.runtime` package holds classes that are
 considered "back-end" components, loaded via our internal CDI implementation
 in order to configure and setup the server.
-
-The `org.wildfly.swarm._CAPABILITY_.deployment` package holds other classes
+* The `org.wildfly.swarm._CAPABILITY_.deployment` package holds other classes
 that should not be considered either part of the front-end API of the fraction
 exposed to users, nor a part of the back-end components to configure the
 server.  Instead, the `.deployment` package is a sidecar to hold additional
 classes that may be added to deployment archives.
+* The `org.wildfly.swarm._CAPABILITY_.detect` package holds detection logic classes.
```

Comment:
Instead of "detection logic", I'd say "fraction autodetection logic", but that's really minor.

Reply:
Sounds much better. Thanks.